open-api/btc/scrypt-open-api.info.mdx

```diff
@@ -0,0 +1,25 @@
+---
+id: scrypt-open-api
+title: "sCrypt Open API"
+description: "Open API Specification of sCrypt Inc."
+sidebar_label: Introduction
+sidebar_position: 0
+hide_title: true
+custom_edit_url: null
+---
+
+import ApiLogo from "@theme/ApiLogo";
+import SchemaTabs from "@theme/SchemaTabs";
+import TabItem from "@theme/TabItem";
+import Export from "@theme/ApiExplorer/Export";
+
+<span className={"theme-doc-version-badge badge badge--secondary"}>Version: v1</span>
+
+<h1 className={"openapi__heading"}>sCrypt Open API</h1>
+
+
+
+Open API Specification of sCrypt Inc.
+
+<div style={{"marginBottom":"2rem"}}><h2 id={"authentication"} style={{"marginBottom":"1rem"}}>Authentication</h2><SchemaTabs className={"openapi-tabs__security-schemes"}><TabItem label={"HTTP: Bearer Auth"} value={"bearerAuth"}><div><table><tbody><tr><th>Security Scheme Type:</th><td>http</td></tr><tr><th>HTTP Authorization Scheme:</th><td>bearer</td></tr><tr><th>Bearer format:</th><td>JWT</td></tr></tbody></table></div></TabItem></SchemaTabs></div><div style={{"display":"flex","flexDirection":"column","marginBottom":"var(--ifm-paragraph-margin-bottom)"}}><h3 style={{"marginBottom":"0.25rem"}}>Contact</h3><span><a href={"mailto:support@scrypt.io"}>support@scrypt.io</a></span></div>
```

Comment:
This part can be placed in the `btc/overview` doc. Remove this doc if necessary.